src/components/ZeroState/ZeroState.tsx

```diff
@@ -45,12 +44,12 @@ export const ZeroState = () => {
             customText='Get started with Insights by adding custom repositories'
             customButton={
               <Button
-                id='add-custom-repositories-button'
-                ouiaId='add_custom_repositories_button'
+                id='get-started-custom-repositories-button'
+                ouiaId='get_started_custom_repositories_button'
```

Comment:
ouiaId and ids both updated @swadeley @mayurilahane ^^

Reply:
just need to make change at this line 
https://gitlab.cee.redhat.com/insights-qe/iqe-content-sources-plugin/-/blob/master/iqe_content_sources/views/repository.py?ref_type=heads#L368